### PR TITLE
refactor: Improve comments 

### PR DESCRIPTION
**Pull request by Ad96el**
## fixes [KILTProtocol/ticket#312](https://github.com/KILTprotocol/ticket/issues/2621)
Clean up comments. 
Rearrange construct_runtime macro.

### CODE REVIEW

File: runtimes/peregrine/src/lib.rs [hunk 1]
```diff
@@ -936,29 +936,27 @@ construct_runtime! {
...
 		TechnicalCommittee: pallet_collective::<Instance2> = 32,
-		// placeholder: parachain council election = 33,
+		// Placeholder: parachain council election = 33,
 		TechnicalMembership: pallet_membership::<Instance1> = 34,
 		Treasury: pallet_treasury = 35,
 		// DELETED: RelayMigration: pallet_relay_migration::{Pallet, Call, Storage, Event<T>} = 36,
 		// DELETED: DynFilter: pallet_dyn_filter = 37,
 
-		// Utility module.
+		// A stateless module with helpers, to allow transmitting batch of calls
```
Comment (ntn-x2) [inline runtimes/peregrine/src/lib.rs:948]: I think we can get rid of this comment altogether. Runtime developers know what this module does.

File: runtimes/peregrine/src/lib.rs [hunk 2]
```diff
@@ -936,29 +936,27 @@ construct_runtime! {
 		Authorship: pallet_authorship::{Pallet, Storage} = 20,
 		AuraExt: cumulus_pallet_aura_ext = 24,
 
-		// Governance stuff
 		Democracy: pallet_democracy = 30,
 		Council: pallet_collective::<Instance1> = 31,
 		TechnicalCommittee: pallet_collective::<Instance2> = 32,
-		// placeholder: parachain council election = 33,
+		// Placeholder: parachain council election = 33,
```
Comment (ntn-x2) [inline runtimes/peregrine/src/lib.rs:942]: Do we need this here? Cc @weichweich

File: runtimes/peregrine/src/lib.rs [hunk 3]
```diff
@@ -936,29 +936,27 @@ construct_runtime! {
...
 		Utility: pallet_utility = 40,
 
 		// Vesting. Usable initially, but removed once all vesting is finished.
 		Vesting: pallet_vesting = 41,
 
-		// System scheduler.
 		Scheduler: pallet_scheduler::{Pallet, Call, Storage, Event<T>} = 42,
 
-		// Proxy pallet.
+		// Allowing accounts to give permission to other accounts to dispatch types of calls from their signed origin
```
Comment (ntn-x2) [inline runtimes/peregrine/src/lib.rs:956]: Same for this. I would say we keep comments only for pallets outside of FRAME, if any.

File: runtimes/peregrine/src/lib.rs [hunk 4]
```diff
@@ -936,29 +936,27 @@ construct_runtime! {
...
 		Proxy: pallet_proxy::{Pallet, Call, Storage, Event<T>} = 43,
 
-		// Preimage registrar
+		// Preimage pallet allows the storage of large bytes blob
```
Comment (ntn-x2) [inline runtimes/peregrine/src/lib.rs:959]: Same.

File: runtimes/peregrine/src/lib.rs (hunk 2 above)
Comment (weichweich) [reply runtimes/peregrine/src/lib.rs:942]: yeah at some point we want to have another election strategy. index 33 is reserved for that. So maybe replace `placeholder` with `reserved`.

File: runtimes/peregrine/src/lib.rs (hunk 1 above)
Comment (weichweich) [reply runtimes/peregrine/src/lib.rs:948]: I think i like it. 😁 Might also be interesting for not so experienced devs.
 
```suggestion
		// A stateless pallet with helper extrinsics (batch extrinsics, send from different origins, ...)
```

**Review by weichweich — COMMENTED** [review]
I think the more verbose comments don't hurt here.

**Review by weichweich — APPROVED** [review]

**Pull request merged**